data/json/mapgen/house_tool_shed.json

```diff
@@ -123,7 +123,7 @@
                 { "item": "fridge", "chance": 20, "x": [ 17,20 ], "y": [ 2,3 ] },
                 { "item": "dining", "chance": 75, "x": [ 18,19 ], "y": [ 2,3 ] },
                 { "item": "magazines", "chance": 50, "x": [ 17,17 ], "y": 5 },
-                { "item": "mischw", "chance": 50, "x": [ 17,21 ], "y": [ 19,21 ] },
+                { "item": "supplies_fuel", "chance": 100, "x": [ 17,21 ], "y": [ 19,21 ], "repeat": [1,2] },
```

Comment:
We should replace mischw and homehw (and maybe hardware) with combination of `tools_home` and `supplies_home` (which would include `supplies_fuel` and such)